go.sum

```diff
@@ -1,2 +1,88 @@
+github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
+github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
```

Comment:
On second thought I'm actually not that happy that this added so many dependencies....

Reply:
Maybe we should push metrics about amount of requests to the boundary to another component and leave this slim